specification/2.0/schema/image.schema.json

```diff
@@ -29,5 +28,8 @@
     "additionalProperties": false,
     "dependencies": {
         "bufferView": [ "mimeType" ]
+    },
+    "not": {
```

Comment:
Only concern for me is code generators. Let me try it out and see what they do with this.